Revolg DOCSUP-998 Document the always_fetch_merged_part setting (#123)
BayoNet/ClickHouse: docs/en/operations/settings/settings.md

Default value: 16.

> ## always\_fetch\_merged\_part {#always_fetch_merged_part}

Reviewer: ```suggestion
## always_fetch_merged_part {#always_fetch_merged_part}
```

> Restricts replica to merge parts by itself.

Reviewer: ```suggestion
Prohibits data parts merging at replicas in [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md)-engine tables.
```

> If enabled, replica never merges parts and always downloads merged parts from other replicas.

Reviewer: I moved this content to the "Possible values" section.

Possible values:

> -   0 — Disabled.

Reviewer: ```suggestion
-   0 — `MergeTree`-engine tables merge data parts at replicas.
```

> -   1 — Enabled.

Reviewer: ```suggestion
-   1 — `MergeTree`-engine tables don't merge data parts at replicas. The tables download merged data parts from other replicas.
```

Default value: 16.

## always_fetch_merged_part {#always_fetch_merged_part}

> Prohibits data parts merging at replicas in [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md)-engine tables.

Reviewer: Let's change `MergeTree` for `ReplicatedMergeTree*` and give a link to https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/replication/ (relative link).

> If enabled, replica never merges parts and always downloads merged parts from other replicas. If there is no merged data yet, the replica waits for them to appear. 

Reviewer: Merge this and the following paragraph into one. Consider starting the united paragraph with the "When merging is prohibited..." phrase instead of "If enabled".

`... **the** replica never ..."

> When the setting is enabled, network load increases and CPU load decreases. This setting can therefore be useful on replicas with relatively weak CPUs or slow disks, such as replicas with archived data.

Reviewer: ```suggestion
CPU and disk load on the replica server decreases, but the network load on cluster increases. This setting can be useful on servers with relatively weak CPUs or slow disks, such as servers for backups storage.
```
